### ISSUE TITLE
local video starts play if open 2 window same time in browser

### ISSUE THREAD

**Issue by alexey2baranov** [andyet/SimpleWebRTC#435]
Hello!

I got another strange thing. If i open 2  windows in browser with same simple example from [](https://github.com/andyet/SimpleWebRTC#its-so-easy) then local video starts play immediatly even if i comment last line with webrtc.joinRoom('your awesome room name');

On the windows machine same test works correct

**Comment by xdumaine** [comment]
I don't fully understand the problem you're describing here. Could you please elaborate?

**Comment by alexey2baranov** [comment]
oh-ho-ho!! my english is bad! I will try again.

I get two diffrenet behavior on linux and windows running the same [demo app from there](https://github.com/andyet/SimpleWebRTC#its-so-easy.) First of all I need understand which one is correct. So can you tell me does local video should start before somebody else connect to my room?

**Comment by fippo** [comment]
yes, the local video starts before someone else enters the room as the default behaviour is to request the camera before connecting.
